roles/ara_api/defaults/main.yaml

```diff
@@ -104,6 +104,9 @@ ara_api_settings: "{{ ara_api_base_dir }}/settings.yaml"
 # ARA_ENV - Environment to load configuration for
 ara_api_env: default
 
+# ARA_EXTERNAL_AUTH - External authentication settings
+ara_api_external_auth: null
```

Comment:
This is a boolean so we should default to false instead of null or none

```suggestion
ara_api_external_auth: false
```

Reply:
Done!